src/_locales/en/messages.json

```diff
@@ -17,6 +17,24 @@
   "continue": {
     "message": "Continue"
   },
+  "errorWalletUnlocked": {
```

Comment:
we probably can enable nested key in i18n config. something like `
"error": {
    "Wallet": {
            Locked": { message": "Wallet is locked"}
      }
  },
`

Reply:
point taken